types/zipcodes/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for zipcodes 6.0
+// Project: https://github.com/davglass/zipcodes
+// Definitions by: Brayden Lopez <https://github.com/headdetect>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface ZipCode {
+  zip: string;
+  latitude: number;
+  longitude: number;
+  city: string;
+  state: string;
+  country: string;
+}
+
+export namespace ZipCodes {
```

Comment:
i do not think this is correct..:
```sh
> var z = require("zipcodes");
undefined
> z.ZipCodes
undefined
```

I think you are missing `export = ZipCodes;`  in this file.

Reply:
Wouldn't that still be the same as `export namespace ZipCodes`, no?